thirdparty/junixsocket/zimbra-junixsocket/debian/control

```diff
@@ -0,0 +1,24 @@
+Source: zimbra-junixsocket
+Build-Depends: debhelper (>= 9), m4, dpkg-dev (>= 1.15.7), maven3
+Section: utils
+Priority: optional
+Maintainer: Zimbra Packaging Services <packaging-devel@zimbra.com>
+Uploaders: Zimbra Packaging Services <packaging-devel@zimbra.com>
+Standards-Version: 3.9.5
+Homepage:  https://github.com/kohlschutter/junixsocket
+
+Package: zimbra-junixsocket
+Section: jars
+Priority: important
+Architecture: any
+Multi-Arch: same
+Depends: ${shlibs:Depends}, ${misc:Depends}, zimbra-base
+Description: junixsocket JARs
+
+Package: zimbra-junixsocket-dbg
+Section: debug
+Priority: extra
+Architecture: any
+Multi-Arch: same
+Depends: ${shlibs:Depends}, ${misc:Depends}, zimbra-base
```

Comment:
Hard to say. To be honest as it is on both OS's the debug packages contain either nothing or nothing useful.

If you know how to get maven to generate useful debug stuff that is compatible with these package managers I'm happy to fix. 

But just doing some basic digging through the POM files and looking over maven I'm not seeing anything that offers any clear clues.

On top of that we are providing a nar file for the libraries and not a traditional library.

So over all to be honest I'm not clear on expectations in this context.

Given all the above I would tend to lean more towards dumping them.

Let me know what direction you want to go and if you want them how you recommend generating what it is you are after.